cms/templates/widgets/user_dropdown.html

```diff
@@ -5,6 +5,9 @@
   from django.urls import reverse
   from django.utils.translation import ugettext as _
   from student.roles import GlobalStaff
+  from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
```

Comment:
Why we are not implementing this file change in the theme?

Reply:
Good catch, Currently we don't have any theme for studio. and as far as the urls of lms are concerned I have to change this PR.

Reply:
Added a todo. Will update this in theme as well when the theme will be in working condition.